generateStuff.py

```diff
@@ -39,6 +39,25 @@
     'OpcUa_Int16'   : 'DPEL_INT'
 }
 
+ObviousMappingArray = {  # these are unanimously good choices
+    'OpcUa_Boolean' : "DPEL_DYN_BOOL",
+    'OpcUa_UInt32'  : "DPEL_DYN_UINT",
+    'OpcUa_Int32'   : "DPEL_DYN_INT",
+    'OpcUa_UInt64'  : "DPEL_DYN_ULONG",
+    'OpcUa_Int64'   : "DPEL_DYN_LONG",
+    'OpcUa_Float'   : "DPEL_DYN_FLOAT",
+    'OpcUa_Double'  : "DPEL_DYN_FLOAT",
+    'UaString'      : "DPEL_DYN_STRING",
+    'UaByteString'  : "DPEL_DYN_BLOB"
+    }
+
+LessObviousMappingArray = {
+    'OpcUa_Byte'    : 'DPEL_DYN_UINT',
+    'OpcUa_SByte'   : 'DPE_DYNL_INT',
```

Comment:
I guess there is a type here, 'DPE_DYNL_INT' should be 'DPEL_DYN_INT'

Reply:
Yes mistake from my side